docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+db:
+  image: mongo
```

Comment:
I think the data persists as long as you don't destroy the container? That at least works w/ an Elasticsearch container i was using.